allure-docker-api/app.py

```diff
@@ -61,8 +62,8 @@
 SWAGGER_URL = '/allure-docker-service/swagger'
 API_URL = '/allure-docker-service/swagger.json'
 SWAGGERUI_BLUEPRINT = get_swaggerui_blueprint(
-    SWAGGER_URL,
-    API_URL,
+    base_url = f'{URL_PREFIX}{SWAGGER_URL}',
```

Comment:
These need to include the prefix so that the UI looks for the js/css/html assets at the correct path. You do not need the prefix when actually registering the blueprint.